src/main/java/ArrayStack.java

```diff
@@ -30,12 +30,13 @@ public ArrayStack(int initialCapacity) {
     }
 
     @Override
-    public void push(T element) {
+    public boolean push(T element) {
```

Comment:
Ah, ah ha. Stacks don't return anything on `push`. They aren't _collections_.

It's weird. I'm not against the return value here, but there is no stack that will actually _return_ `false` here.

Reply:
in that case, ctrl-z.

Reply:
Upon further thought, I will leave it to return a boolean.